packages/flutter/lib/src/material/icons.dart

```diff
@@ -38,7 +38,7 @@ class IconData {
 ///
 ///  * [Icon]
 ///  * [IconButton]
-///  * <https://design.google.com/icons/>
+///  * [design.google.com/icons](https://design.google.com/icons/)
```

Comment:
Sounds good! There's a customer-flutter tag in that repo, which would be worth using.